src/HasParent.php

```diff
@@ -146,6 +146,21 @@ protected function getParentClass(): string
     {
         static $parentClassName;
 
-        return $parentClassName ?: $parentClassName = (new ReflectionClass($this))->getParentClass()->getName();
```

Comment:
`$parentClassName` was set to static for performance reasons.
This update returns `$parentClassName` if it is set, but never sets it which could have performance implications. 

See my related [comment](https://github.com/tighten/parental/pull/104#discussion_r1220586401) from PR #104.